spec/definitions/CustomerInformation.yaml

```diff
@@ -0,0 +1,21 @@
+type: object
+properties:
+  currency:
+    description: Currency (three letter ISO 4217 alpha code) (eg USD, EUR)
+    readOnly: true
+    type: string
+  refunds:
```

Comment:
I suggest `refundsAmount`, `disputesAmount` as at some point in the future we may want to add counts.